### ISSUE TITLE
Content Security Policy causes web font loading errors on Chrome 66

### ISSUE THREAD

**Issue by ElectronicWar** [mozilla/send#804]
Running Chrome Version 66.0.3359.81 (Official Build) beta (64-bit) on Windows 10.

Chrome is the only browser that is reporting web font loading errors due to the CSP set by Send.
Error is solved when adding `data:` to the `font-src` CSP part.

Other tested browsers show no messages in the log (Firefox 60.0b11 and Edge 41), they 
It seems that Chrome is very strict in applying the CSP.

Example:
![image](https://user-images.githubusercontent.com/1890874/38642383-3434b698-3dda-11e8-967f-3a89fa18b14a.png)

**Comment by ElectronicWar** [comment]
The problem still present for me on send.firefox.com

I am pretty sure this cannot be fixed by the CDN as the send node app is setting the CSP headers.

**Comment by dannycoates** [comment]
> I am pretty sure this cannot be fixed by the CDN as the send node app is setting the CSP headers.

True, however the css file on the cdn shouldn't be using `data` anymore (https://code.cdn.mozilla.net/fonts/fira.css). I do not get any CSP errors on Chrome 66 with a clean cache.

**Comment by ElectronicWar** [comment]
A hard reload with empty cache still returns the errors for me, but interestingly not anymore when using an incognito tab.
Seems like I get served an older cached version (maybe based on a cookie). I will re-check periodically, but this can be closed for good I guess.
